### PR TITLE
tests: added for info cmd

### PR DESCRIPTION
**Pull request by niravasher**
**What kind of change does this PR introduce?**
Test

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**
Added tests for config #1012

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by niravasher** [comment]
@anikethsaha please suggest changes here.

File: test/info/info.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { sep } = require('path');
+const { run, extractSummary } = require('../utils/test-utils');
+const pkgJSON = require('../../package.json');
+
+describe('info flag', () => {
+    it('outputs version of webpack', () => {
```
Comment (rishabh3112) [inline test/info/info.test.js]: This test isn't aimed at `info` command. Version is provided by `--version` flag.

File: test/info/info.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        const result1 = str.slice(1);
+        expect(pkgJSON.dependencies.webpack).toContain(result1);
+        expect(stderr).toContain('option has not been set, webpack will fallback to');
+        const outputDir = 'info/bin';
+        const outDirectoryFromCompiler = summary['Output Directory'].split(sep);
+        const outDirToMatch = outDirectoryFromCompiler.slice(outDirectoryFromCompiler.length - 2, outDirectoryFromCompiler.length).join('/');
+        expect(outDirToMatch).toContain(outputDir);
+    });
+
+    it('outputs versions with dashed syntax', () => {
```
Comment (rishabh3112) [inline test/info/info.test.js]: `info` is not a flag but a command.

**Comment by niravasher** [comment]
Info gives information of Binaries, system, browsers and npmPackages where all npm packages will involve a version check for wepack. So Ill add node and npm versions. Is that right? @rishabh3112

**Comment by rishabh3112** [comment]
Not exactly, Try experimenting with `info` package to understand what it do. Here is some starting steps for your help:
1. Create a empty folder out of your fork.
2. Run following commands:
```bash
npm init
npm install --save-dev webpack@next webpack-cli@next @webpack-cli/init
```
3. Access info command:
```
npx webpack info
```

**Comment by niravasher** [comment]
I did this already. This shows the following
Screenshot from 2020-01-09 23-39-29
@rishabh3112

**Comment by rishabh3112** [comment]
Just go ahead and test for `info`, not `--info`.

**Comment by niravasher** [comment]
```
it('outputs versions with dashed syntax', () => {
    const { stderr } = run(__dirname, ['--info']);
        expect(stderr).toContain('option has not been set, webpack will fallback to');
    });
});
```
U mean remove this part and update my PR? @rishabh3112

**Comment by rishabh3112** [comment]
Yup and the test above it too.

**Comment by niravasher** [comment]
```
const { stdout, stderr } = run(__dirname, ['info']);
console.log(stdout);

```
gives
Screenshot from 2020-01-12 12-08-07
@rishabh3112 . How should I continue?

**Comment by anshumanv** [comment]
You need to symlink the package to use the info package present locally, refer the contributing docs.

**Comment by niravasher** [comment]
> You need to symlink the package to use the info package present locally, refer the contributing docs.

By typing npm link, yes done that already @anshumanv

**Comment by anshumanv** [comment]
It's not done correctly then, otherwise, your CLI code will execute the code from the info package locally, can you try linking again?

**Comment by niravasher** [comment]
I am in my project folder
I run the following cmds
npm link
npm link webpack-cli
npm run bootstrap
npm run build
npx jest test/info/info.test.json
I get the same output, am I missing anything? @anshumanv

**Comment by niravasher** [comment]
Ill reclone the project

**Pull request closed without merging**